code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -621,6 +621,33 @@
 	if(need_mob_update)
 		return UPDATE_MOB_HEALTH
 
+/datum/reagent/medicine/c2/penthrite/proc/send_alert(mob/living/affected_mob)
+	affected_mob.throw_alert("penthrite", /atom/movable/screen/alert/penthrite)
+
+/datum/reagent/medicine/c2/penthrite/proc/remove_alert(mob/living/affected_mob)
+	affected_mob.clear_alert("penthrite")
+
+///Alternative to penthrite that keeps you up for a few seconds after having a heart attack. Gives a bit of time to call for help regardless of when/where you've collapsed.
+/datum/reagent/medicine/c2/penthrite/heart_attack
```

Comment:
Make sure this is not synthable, ie botany can't get it

(Tho I think it's a little goofy to use a reagent for this anyways)